src/components/Application/BriefOwnerHeader.tsx

```diff
@@ -56,6 +68,55 @@ const BriefOwnerHeader = (props: BriefOwnerHeaderProps) => {
         setAnchorEl(null);
     };
 
+    const getBalance = useCallback(async (walletAddress: string, currency_id: number) => {
```

Comment:
return type missing

Reply:
This might get complicated since it is an async function and async function returns promise but we just want the balance as string